catroid/src/main/res/values/strings.xml

```diff
@@ -131,6 +131,8 @@
     <string name="preference_description_nfc_bricks">Allow to use NFC tags in programs</string>
     <string name="preference_description_crash_reports">Allow to send crash reports automatically</string>
     <string name="preference_title_enable_crash_reports">Send anonymous crash reports</string>
+    <string name="preference_description_cloud_notifications">Allow to receive cloud notifications</string>
+    <string name="preference_title_enable_cloud_notifications">Enable Cloud Notifications</string>
```

Comment:
Please swap these lines.
1. title
2. description

Please create a PR to correct the order of the crash_reporter strings as well.